src/components/Clusters/views/EditCluster/AuthenticationDropdown.js

```diff
@@ -14,7 +14,7 @@ export function AuthenticationTypeDropdown({ type, setType }) {
       options={options}
       selectedKey={type}
       onSelect={(_, selected) => setType(selected.key)}
-      placeholder={t('issuers.placeholders.type')}
+      placeholder={t('clusters.placeholders.type')}
```

Comment:
It's unnecessary, can be removed

Reply:
sorry what exactly do you mean, just line 17 ?